From 137738de2c53d4b077022c27af9085b0b7f26cfd Mon Sep 17 00:00:00 2001
From: "bo.yang" <bo.yang@amlogic.com>
Date: Fri, 30 May 2014 12:01:29 +0800
Subject: [PATCH 4265/5965] m6tvc: use dtd to config wifi power gpio

requested by kieth.liu@amlogic.com
reviewed  by mengchun.li@amlogic.com
---
 arch/arm/boot/dts/amlogic/meson6tvc_h15.dtd      | 1 +
 arch/arm/boot/dts/amlogic/meson6tvc_h15_512m.dtd | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/amlogic/meson6tvc_h15.dtd b/arch/arm/boot/dts/amlogic/meson6tvc_h15.dtd
index 3969489bfaff..c5f852c01eb9 100755
--- a/arch/arm/boot/dts/amlogic/meson6tvc_h15.dtd
+++ b/arch/arm/boot/dts/amlogic/meson6tvc_h15.dtd
@@ -643,6 +643,7 @@
 		dev_name = "wifi_power";
 		status = "okay";
 		power_gpio = "GPIOW_19";
+		valid = "low";
 	};
 
 	gpio:gpio {
diff --git a/arch/arm/boot/dts/amlogic/meson6tvc_h15_512m.dtd b/arch/arm/boot/dts/amlogic/meson6tvc_h15_512m.dtd
index 4c67b3d6f9f2..92810815d316 100755
--- a/arch/arm/boot/dts/amlogic/meson6tvc_h15_512m.dtd
+++ b/arch/arm/boot/dts/amlogic/meson6tvc_h15_512m.dtd
@@ -632,6 +632,7 @@
 		dev_name = "wifi_power";
 		status = "okay";
 		power_gpio = "GPIOW_19";
+		valid = "low";
 	};
 
 	gpio:gpio {
-- 
2.19.0

